### ISSUE TITLE
Unknown bug on starting MobileNode

### ISSUE THREAD

**Issue by ArtemHryhorovGeniusee** [mysteriumnetwork/node#3687]
This happened right after app was launched
Google Pixel 3, Android 11
I attached the file with all Go logs 
[mysterium_log.txt](https://github.com/mysteriumnetwork/node/files/6884445/mysterium_log.txt)

**Comment by Snawoot** [comment]
Related fragment from runtime source:

```
func mapassign_faststr(t *maptype, h *hmap, s string) unsafe.Pointer {

	if h == nil {

		panic(plainError("assignment to entry in nil map"))

	}

	if raceenabled {

		callerpc := getcallerpc()

		racewritepc(unsafe.Pointer(h), callerpc, funcPC(mapassign_faststr))

	}

	if h.flags&hashWriting != 0 {

		throw("concurrent map writes")

	}

	key := stringStructOf(&s)

	hash := t.hasher(noescape(unsafe.Pointer(&s)), uintptr(h.hash0))
```

It seems it panics on `throw("concurrent map writes")`. We should either protect map with mutex or ensure there are no concurrent use of Config instance.

**Comment by Snawoot** [comment]
This crash was fixed via #3744. However, in stack trace I see two `mysterium.NewNode` invocations with different arguments:

```
    github.com/mysteriumnetwork/node/mobile/mysterium.NewNode(0x40001500c0, 0x28, 0x4000198000, 0x28, 0x400009a058, 0x6f78193f78)
    github.com/mysteriumnetwork/node/mobile/mysterium.NewNode(0x4000168180, 0x28, 0x4000176300, 0x28, 0x400009a058, 0x6f78193f78)
```

It indicates at least two node instances were created. Such use case may lead to unexpected behavior of MobileNode because it uses shared resources a lot.
